### PR TITLE
Fix: Keep packages sorted

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] keeps packages in `composer.json` sorted without having to specify `--sort-packages` when requiring packages

Also not directly related to #433.

💁‍♂️ For reference, see https://getcomposer.org/doc/06-config.md#sort-packages.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
     "license": "MIT",
+    "config": {
+        "sort-packages": true
+    },
     "require": {
         "php": ">=5.5",
-        "twig/twig": "1.*",
-        "pagerfanta/pagerfanta": "1.0.*@dev",
+        "aptoma/twig-markdown": "0.2.*",
         "cartalyst/sentry": "2.1.*",
```
Comment (localheinz) [inline composer.json]: I added the `config` section above, then I ran

```
$ composer require "cartalyst/sentry:2.1.*"
```

to trigger the sorting, as this is the only package that isn't outdated.

```
$ composer info --direct | grep cartalyst/sentry
cartalyst/sentry              v2.1.6             PHP 5.3+ Fully-featured Authentication & Authorization System

$ composer outdated | grep cartalyst/sentry

Package fabpot/php-cs-fixer is abandoned, you should avoid using it. Use friendsofphp/php-cs-fixer instead.
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
```

**Comment by chartjes** [comment]
@localheinz There's a conflict with the composer.lock file that needs to be resolved...

**Comment by localheinz** [comment]
@chartjes 

Let me sort this out once #434 has been merged!

**Comment by localheinz** [comment]
Rebased!

**Comment by localheinz** [comment]
Thanks for reviewing all the PRs, @chartjes!

**Pull request merged**